WidgetExtension/WidgetEnvironment.swift

```diff
@@ -15,8 +15,8 @@ import UIKit
 final class WidgetEnvironment {
     static let shared = WidgetEnvironment()
     
-    var server: Server?
-    var user: SignedInUser?
+    var server: Server!
+    var user: SignedInUser!
```

Comment:
Widgets can be created and accessed without a server and user.
This can cause a lot of crashes.
It would be better to explicitly declare this as "?".

Reply:
93edcee7dd906649d9104ace65152af388eed830
resolved